ledger/sandbox/src/main/scala/com/digitalasset/ledger/server/apiserver/ApiServices.scala

```diff
@@ -12,21 +12,21 @@ import com.daml.ledger.participant.state.index.v2.{
   _
 }
 import com.daml.ledger.participant.state.v2.WriteService
+import com.daml.ledger.participant.state.v1.TimeModel
```

Comment:
Use `v2` of the participant api. Ideally also remove the dependency on v1 altogether, like we did for the index-api v2.